src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,26 @@
+package calculator;
+
+public enum Operator {
+    ADD("+"),
+    SUBTRACT("-"),
+    MULTIPLY("*"),
+    DIVIDE("/");
+
+    private final String symbol;
+
+    Operator(String symbol) {
+        this.symbol = symbol;
+    }
+
+
+    public static Operator findBySymbol(String symbol) {
+
+        for (Operator value : values()) {
+            if (value.symbol.equals(symbol)) {
+                return value;
+            }
+        }
+        throw new IllegalArgumentException("[ERROR] 연산자는 +, -, * , / 만 가능합니다.");
+    }
```

Comment:
`try catch`문이 없어 의도한대로 exception을 던져주지 못할 것 같아요 ″̮

https://cheershennah.tistory.com/147

```suggestion
public static Operator findBySymbol(String symbol) {
        return Arrays.stream(values()).filter(val -> val.symbol.equals(symbol)).findFirst().orElseThrow(IllegalArgumentException::new);
    }
```
이렇게 가능하니 참고 부탁드립니다 ″̮